axol/axol.py

```diff
@@ -2093,6 +2093,23 @@ def love_mute(message):
         return True
 
 
+caught = False
+
+
+def catch(message):
+    if message.chat.id != vip_chat_id:
+        return
+    global caught
+    if caught:
```

Comment:
You could use next message as input with ` bot.register_next_step_handler`
See for example:
https://github.com/eternnoir/pyTelegramBotAPI/blob/888c7a6b0d97c376947ee7b525d4b70583acc3fc/examples/step_example.py

Reply:
If we were on Stack Overflow, I'd upvote this advice.